README.md

```diff
@@ -101,12 +101,12 @@ extension UIImage {
     case BigPear = "Big_Pear"
     case StopButtonEnabled = "stop.button.enabled"
 
-    var image: UIImage {
-      return UIImage(named: self.rawValue)!
+    var image: UIImage? {
+      return UIImage(named: self.rawValue)
     }
   }
 
-  convenience init!(asset: Asset) {
+  convenience init?(asset: Asset) {
```

Comment:
Unfortunately, I'm not sure if there's actually a way to make use of the fatalError here. The example you have won't compile because a non-failable initializer can't delegate to a failable initializer. 

One option would be to make the asset initializer failable and use the fatalError in the image property on the Asset enum:

```
extension UIImage {
    enum Asset: String {
        case Logo = "logo"
        
        var image: UIImage {
            guard let image = UIImage(asset: self) else {
                fatalError(
                    "You used an Asset enum which doesn't match an existing Image. \n"
                        + "Be sure to run SwiftGen again to ensure the enum and the Assets Catalog are in sync"
                )
            }
            return image
        }
    }
    
    convenience init?(asset: Asset) {
        self.init(named: asset.rawValue)
    }
}
```

Of course, with this route, you'd still be stuck having to use an optional UIImage if you call this initializer directly.

Do you know of any other way you could make this work?